syncapi/routing/messages.go

```diff
@@ -41,19 +41,17 @@ type messagesReq struct {
 	roomID           string
 	from             *types.TopologyToken
 	to               *types.TopologyToken
-	fromStream       *types.StreamingToken
 	device           *userapi.Device
 	wasToProvided    bool
 	backwardOrdering bool
 	filter           *gomatrixserverlib.RoomEventFilter
 }
 
 type messagesResp struct {
-	Start       string                          `json:"start"`
-	StartStream string                          `json:"start_stream,omitempty"` // NOTSPEC: so clients can hit /messages then immediately /sync with a latest sync token
```

Comment:
Cerulean uses this to load quickly. https://github.com/matrix-org/cerulean/blob/main/src/Client.js#L401